doc/classes/TextEdit.xml

```diff
@@ -404,10 +404,10 @@
 			If [code]true[/code], read-only mode is enabled. Existing text cannot be modified and new text cannot be added.
 		</member>
 		<member name="scroll_horizontal" type="int" setter="set_h_scroll" getter="get_h_scroll" default="0">
-			The current horizontal scroll value.
+			If there is a horizontal scrollbar this determines how many pixels from the left the text is.
```

Comment:
"horizontal scroll value" is probably descriptive enough and only misses the information that it uses pixels.
What about something like "If there is a horizontal scrollbar this determines the current horizontal scroll value in pixels"?